README.md

```diff
@@ -1,2 +1,46 @@
 # beer-is-poetic
 Beer is Poetic
+
+## Description
+
+This website allows you to search for a brewery when you enter a city name and will generate a satellite image of the city as well as a random poem that has the word "beer" in it and credits the author.
+
+3 APIs were called:
+    List of breweries by city
+    Satellite image of city
+    Random poems
+
+Built on: 
+    HTML 
+    CSS 
+    JS
+Tailwinds for bootstrap
```

Comment:
Maybe put links to tailwind, jquery and jquery documentation. 
For tailwind, mention that for purposes of this project we were reuired to use the CDN interface (not recommended by Tailwind for live sites, just testing) and will use installation in future.